.travis.mk

```diff
@@ -159,7 +170,7 @@ test_coverage_debian_no_deps: build_coverage_debian
 	# coveralls API: https://docs.coveralls.io/api-reference
 	@if [ -n "$(COVERALLS_TOKEN)" ]; then \
 		echo "Exporting code coverage information to coveralls.io"; \
-		gem install coveralls-lcov; \
+		sudo gem install coveralls-lcov; \
```

Comment:
This one also looks as the command for `*_deps` goal.

Reply:
Right, corrected.